torchvision/csrc/cpu/DeformConv_cpu.cpp

```diff
@@ -713,55 +713,49 @@ static std::tuple<at::Tensor, at::Tensor> deform_conv2d_backward_input_cpu(
 
   auto grad_input = at::zeros_like(input);
   auto grad_offset = at::zeros_like(offset);
-  auto columns = at::zeros(
+  auto columns = at::empty(
       {n_in_channels * weight_w * weight_h, n_parallel_imgs * out_h * out_w},
       input.options());
 
   // Separate into blocks
-  grad_input = grad_input.view(
+  grad_input = grad_input.reshape(
```

Comment:
Why is this changing from view to reshape?

Reply:
There are many changes like that which are mostly artifacts of when I was debugging ongoing issues, and although most of them are not needed per se, I think it's a better practice now to use `reshape` instead of `view`, as it works with non-contiguous tensors.

Basically, using `reshape` here will probably not change anything to the current code-path, but I think it's safer

Reply:
It's safer, but it can also incur unexpected memory operations

Reply:
totally agree, but if we want to support non-contiguous tensors in this function, we would need to call `.contiguous()` before anyway, so this becomes a no-op